client/blocks/importer/wordpress/upgrade-plan/test/index.tsx

```diff
@@ -56,11 +45,52 @@ const DEFAULT_SITE_CAPABILITIES = {
 	promote_users: true,
 	publish_posts: true,
 	remove_users: true,
+	update_plugins: true,
 	upload_files: true,
 	view_hosting: true,
 	view_stats: true,
 };
 
+// Stub out UpgradePlanDetails because it has much more complex dependencies, and only provides a wrapper around the content from this component.
+jest.mock( '../upgrade-plan-details', () => ( {
```

Comment:
Nit:
Was there a special reason to move the mocks after the constants and add the other mocks before the constants? Could we keep one block of code for mocks and another for the constants to make it more organized?

Reply:
Updated here: https://github.com/Automattic/wp-calypso/pull/92201/commits/edff35cbaf5b91c6ee6cfa3275d5a17b3d2ad7c0